verif/MeetingMinutes/Verification Issues Meeting Nov4 2020.md

```diff
@@ -0,0 +1,89 @@
+Meeting report - core-v-verif issues
+Attendees: Duncan Bees, Mike Thompson, Steve Richmond, Arjan Bink, Greg Tumbush, John Martin, Lee Moore
+The purpose of the meeting was to review open issues on core-v-verif and attempt to resolve them
+Scribe: Duncan Bees
+
+
+Issue #47
+-	Issue: Update the XPULP compliance testsuite into the uvmt_cv32 CV32E40P verification environment
+-	Waived for RTL Freeze obvious reason - PULP features not supported in this release.
+-	General discussion about labeling strategy for Github issues. Are the labels associated with a tag, i.e. will labels made at the time of a tagged 
+-       release stay as they are?  
+-	(In cores repo, a different label system was used. A "PULP" label is used for "PULP" features. This is potentially more permanent.
+-	AI: Mike will find out what happens to issues on a tag
+-	AI: Mike, when we do tag for RTL Freeze, we should have a document to describe issues
+
+
+#48
+-	Similar to 47, this one is functional coverage – same waiving strategy as 47
+
+#50 – 	Task: ensure 100% coverage of ISA instructions
+-	This issue will be closed once functional coverage is 100% complete

```

Comment:
...once ISA functional coverage is 100%...

Reply:
fixed